src/registrar/models/domain.py

```diff
@@ -909,11 +909,22 @@ def renew(self):
         """Time to renew. Not implemented."""
         raise NotImplementedError()
 
-    def get_security_email(self):
-        logger.info("get_security_email-> getting the contact ")
-        secContact = self.security_contact
-        if secContact is not None:
-            return secContact.email
+    def get_security_email(self, skip_epp_call=False):
+        logger.info("get_security_email-> getting the contact")
+
+        # If specified, skip the epp call outright.
+        # Otherwise, proceed as normal.
+        if skip_epp_call:
+            logger.info("get_security_email-> skipping epp call")
+            security = PublicContact.ContactTypeChoices.SECURITY
+            security_contact = self.generic_contact_getter(security, skip_epp_call)
+        else:
+            security_contact = self.security_contact
+
```

Comment:
Reasoning for this change:

In the original csv_export ticket, a workaround was used to query the value of the security email, but not trigger the EPP call. This was inefficient as it meant we had to do a DB call everytime we wanted this value, rather than relying on a single one and pulling from that cached data.

Rather than doing that, we can just modify the behavior of this function to specify this. This allows us to selectively toggle that behavior for scenarios where we might not want it (for example, a csv export or a script). By default, it will always pull from EPP

Reply:
It is gone!